style.css

```diff
@@ -1,3 +1,287 @@
 * {
   box-sizing: border-box;
+  /*border: 1px solid red;*/
+}
+
+body {
+  margin: 0 auto;
+}
+
+.announcementbar {
+  margin: 0;
+  width: 100%;
+  background: #fa6938;
+  padding: 0.01rem;
+  font-family: 'Source Sans Pro', sans-serif;
+  color: black;
+}
+
+.announcementbar p {
+  margin-left: 5rem;
+  color: white;
+}
+
+#shipping {
+  color: white;
+  background: rgba(0,0,0,0.2);
+  border-radius: 0.5rem;
+  padding: 0.4rem;
+  text-transform: uppercase;
+}
+
+.top-nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.site-logo {
+  height: 8rem;
+  width: auto;
+  margin-left: 5rem;
+}
+
+.nav-list {
+}
+
+.accountlinks {
+  font-family: 'Source Sans Pro', sans-serif;
+  color: gray;
+  text-align: right;
+  margin-right: 5rem;
+  padding: 0.2rem;
+  /*text-decoration: none;*/
+}
+
+.signin {
+  text-decoration: none;
+  color: gray;
+}
+
+.container {
+  display: block;
+}
+
+.InputAddOn {
+  float: left;
+}
+
+.InputAddOn-field {
+  flex: 1;
+}
+
+.InputAddOn-item {
+}
+
+.accountsearch {
+  margin-right: 5rem;
+  position: relative;
+}
+
+.accountsearch button {
+  float: left;
+  border-radius: 0.3rem;
+  padding: 0.2rem;
+  margin: 0.2rem;
+}
+
+.nav-list__item {
+  list-style: none;
+  display: inline;
+  padding: 0.5rem;
+  font-family: 'Source Sans Pro', sans-serif;
+  color: black;
+  font-size: 1.5rem;
+  font-weight: lighter;
+}
+
+
+.fa-search {
+  color: gray;
+}
+
+
+.hero-container {
+  display: block;
+  background: #F5F5F5;
+  border-top-style: groove;
+}
+
+.tieselection {
+  display: flex;
+  justify-content: center;
+  margin: auto;
+  list-style: none;
+  width: 80%;
+  padding-left: 0;
+}
+
+.tie img {
+  width: 100%;
+}
+
+.tie {
+  text-decoration: none;
+}
+
+.tiecolor {
+  text-align: center;
+  font-family: 'Source Sans Pro', sans-serif;
+  color: black;
+}
+
+.tagline {
+  display: block;
+  text-align: center;
+  padding: 1rem;
+  font-family: 'Source Sans Pro', sans-serif;
+  font-size: 4rem;
+}
+
+.three {
+  display: flex;
+  width: 80%;
+  justify-content: center;
+  margin: auto;
+}
+
+.class {
+  width: 100%;
+}
+
+.three-img {
+  margin: 1rem;
+}
+
+.three-img p {
+  font-family: 'Source Sans Pro', sans-serif;
+  font-size: 1.3rem;
+  font-weight: bold;
+  transform: translate(12rem, -11rem);
```

Comment:
If you use `position: absolute`, you can pin things perfectly to the upper-right hand side of the object.
<img width="790" alt="screen shot 2016-06-22 at 2 06 31 pm" src="https://cloud.githubusercontent.com/assets/2532004/16279869/8c25cb08-3882-11e6-823d-51efdc2e5672.png">